crates/core/plugin_sm/src/plugin.rs

```diff
@@ -461,13 +468,54 @@ impl Plugin for ExternalPluginCommand {
         let command = self.command(LIST, None)?;
         let output = self.execute(command, logger).await?;
         if output.status.success() {
+            let filtered_output = match (&self.exclude, &self.include) {
+                (None, None) => output.stdout,
+                (Some(exclude), Some(include)) => {
+                    let exclude_filter = Regex::new(format!(r"^{}\t.*", exclude).as_str())?;
```

Comment:
suggestion: We tell the user that these options are regexes, will the code correctly handle when the user includes `^`, `$`, `\t` or `.*` as part of their regex? I'd like to see it in the test case to make sure.

Reply:
As a user I would also find the underlying behaviour unexpected, as I can't could do a "starts-with" filter using `^python3-`.

I might make more sense to run the filter on the **package name** field only (e.g. split the string first on `\t` and run the regex on the first element). Then the user can use the full regex syntax, e.g. `^(foo|bar)$`

Reply:
Added extra split so now only package name is filtered.